### DISCUSSION TITLE
Embed component for modal

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12671] [Help] [answered]
I tried to resize a modal for logout feature using embed component, I plan to update state when I click the logout button, but the problem is dynamic filter don't have any on-action option to update the state.. is there any way to add an action when I click the dynamic filter? or any other advice to achieve my purpose?

I'm using self-hosted, and version 2.13.9

![e959a65e-4cde-4690-a7bb-3f65757ab63c](https://github.com/Budibase/budibase/assets/134992316/6f621b53-548b-4341-9623-ba6400b4f2ff)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @fadliasyhariStar,

This component isn't configured for using/updating state, therefore the answer to your question is no currently. If you are wanting to have a more permanent solution you may be better off using a different field component and managing/setting the state for/from this.

You could create a feature request [here](https://github.com/Budibase/budibase/issues/new/choose), if you feel this is something which needs added to the core product.
